bower.json

```diff
@@ -27,18 +27,15 @@
     "rjs.noop.js"
   ],
   "dependencies": {
-    "requirejs": "2.1.17",
-    "requirejs-plugins": "1.0.3",
-    "require-css": "0.1.8",
-    "c3": "0.4.9",
-    "colorbrewer": "1.0.0",
-    "d3": "3.5.0",
-    "dagre": "0.7.1",
```

Comment:
No, I don't want any particular version. It's fine to include another version if it's in a git repo.